pkg/console/subresource/configmap/configmap_builder.go

```diff
@@ -0,0 +1,156 @@
+package configmap
+
+import (
+	operatorv1 "github.com/openshift/api/operator/v1"
+	"github.com/openshift/console-operator/pkg/api"
+	yaml "gopkg.in/yaml.v2"
+	"k8s.io/klog"
+)
+
+// ConsoleYamlConfigBuilder
+// Director will be DefaultConfigMap()
+//
+// b := ConsoleYamlConfigBuilder{}
+// return the default config value immediately:
+//   b.Config()
+// set all the values:
+//   b.SetHost(host).SetLogoutURL().setBrand().setDocURL().SetAPIServerURL().SetProductName().SetCustomLogoFile().Config()
```

Comment:
The method names here are not reflecting the method names from the `YAMLConfigBuilder` interface below

Reply:
Removed the interface as its not needed.